mirrulations-core/src/mirrcore/redis_check.py

```diff
@@ -0,0 +1,7 @@
+import redis
+
+def is_redis_available(database):
+        try:
+            return database.ping()
+        except (ConnectionRefusedError, redis.BusyLoadingError):
+            return False
```

Comment:
I think this function needs to be tabbed back once.

Reply:
Good catch, fixed!